packages/core/modules-sdk/src/medusa-app.ts

```diff
@@ -379,24 +389,53 @@ async function MedusaApp_({
         }
       }
 
-      await MedusaModule.migrateUp(
-        moduleResolution.definition.key,
-        moduleResolution.resolutionPath as string,
-        moduleResolution.options,
-        moduleResolution.moduleExports
-      )
+      if (!revert) {
```

Comment:
```suggestion
      if (revert) {
```
And then change the order they are performed. Optionally, and better IMO, is to rename the variable to `apply` instead of the negative form `revert`

Reply:
do you mean, apply to specify the run migration and if it is false then I revert?